seed_data/challenges/computer-science.json

```diff
@@ -21,8 +21,13 @@
       "descriptionFr": [],
       "nameRu": "",
       "descriptionRu": [],
-      "nameEs": "",
-      "descriptionEs": [],
+      "nameEs": "Waypoint: Aprende Informática Básica",
+      "descriptionEs": [
+        "Stanford tiene un excelente currículo de Informática. Este curso interactivo utiliza una versión modificada de JavaScript. Este curso cubrirá rapidamente varias de las ideas.",
```

Comment:
Hi, I would include the word "gratuíto" (=free) and "en Internet" (~online) in the first phrase, so I would have translated it like this: "Stanford tiene en internet un excelente currículo gratuíto sobre Informática."

I also wouldn't repeat "curso" in the same paragraph. In the last phrase I would say "En él se cubrirán varios conceptos rápidamente."